nautobot/dcim/filters/__init__.py

```diff
@@ -866,7 +869,9 @@ class DeviceFilterSet(
         to_field_name="name",
         label="Rack (name or ID)",
     )
+    # Even though devices can be assigned to multiple clusters, we only want to filter by one.
     cluster = NaturalKeyOrPKMultipleChoiceFilter(
+        field_name="clusters",
```

Comment:
I think we should add a `clusters` filter as well (pretty much just a duplicate of this one) and consider the `cluster` filter to be deprecated. Just a matter of self-consistency.